orca-sql/src/test/groovy/com/netflix/spinnaker/orca/sql/cleanup/OldPipelineCleanupPollingNotificationAgentSpec.groovy

```diff
@@ -95,7 +99,7 @@ class OldPipelineCleanupPollingNotificationAgentSpec extends Specification {
     Execution e = new Execution(PIPELINE, "app")
     e.status = ExecutionStatus.SUCCEEDED
     e.pipelineConfigId = "pipeline-001"
-    e.buildTime = (new Date() - daysOffset).time
+    e.buildTime = Instant.now().minus(daysOffset, DAYS).toEpochMilli()
```

Comment:
Groovy's operators for `Date` are in an optional jar now. Easier to use a better type.